src/content/docs/kubernetes-pixie/auto-telemetry-pixie/advanced-configuration/manage-pixie-data.mdx

```diff
@@ -8,13 +8,17 @@ tags:
   - Kubernetes pods
   - Kubernetes
   - manage Pixie data
-metaDescription: 
+metaDescription:
 redirects:
   - /docs/auto-telemetry-pixie/manage-pixie-data
 ---
 
 You have options for the type and the amount of data you ingest when you install Auto-telemetry with Pixie. During install, use Helm to reduce, restrict, or exclude data, by excluding specific namespaces or pods, by collecting data for only the nodes you want, or by redacting columns that contain sensitive information.
 
+<Callout variant="important">
+The following sections show how to configure reduced ingest during installation of Pixie. You can also reduce ingest after installation by [disabling select Pixie export scripts](/docs/kubernetes-pixie/auto-telemetry-pixie/understand-use-data/overview/#pixie-configure-storage).
```

Comment:
i would emphasize configure over reduce here. people might want to add new export scripts